packages/components/app/styles/components/copy/button.scss

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// COPY > BUTTON
+//
+
+// Colors + States
+
+.hds-button--color-secondary {
+  .hds-button__icon {
+    color: var(--token-color-foreground-action);
+  }
+
+  &.is-success,
+  &.mock-success {
```

Comment:
I thought our preference for classes like these was to use BEM style naming to scope them.

Reply:
since this is what we used in copy-item in the dropdown, I kept it the same. Do you think it's safe to change?

Reply:
I don't see why it wouldn't be safe to change. I'd worry more about one of the products using a classname like "is-success" and inadvertently changing the style of the HDS components.

Reply:
ok, changed it.